tests/datasets.py

```diff
@@ -109,6 +109,14 @@ class DATASET_OPTIONS_T(TypedDict):
             'derivatives/freesurfer/subjects/sub-01/mri/aparc.a2009s+aseg.mgz'
         ],
     },
+    'ds000248_no_mri': {
+        'git': '',
+        'openneuro': 'ds000248',
+        'osf': '',
+        'web': '',
+        'include': ['derivatives/freesurfer/subjects/sub-01'],
+        'exclude': ['derivatives/freesurfer'],
```

Comment:
can you also exclude the "anat" folder?

Reply:
The current combination of include and exclude that I've added here actually doesn't make sense … We only need `derivatives/freesurfer`, right?? Will update